File: app/src/main/java/com/bq/zowi/models/commands/GridCommand.java
package com.bq.zowi.models.commands;

/* JADX INFO: loaded from: classes.dex */
public class GridCommand {
    private Command command;
    private boolean unlocked = true;

    public GridCommand(Command command) {
        this.command = command;
    }

    public Command getCommand() {
        return this.command;
    }

    public boolean isUnlocked() {
        return this.unlocked;
    }

    public void setUnlocked(boolean unlocked) {
        this.unlocked = unlocked;
    }

    public String getCommandId() {
        return getCommand().getId();
    }
}

File: zowi-core/src/main/kotlin/com/bq/zowi/models/commands/GridCommand.kt
package com.bq.zowi.models.commands

class GridCommand(private val command: Command) {
    private var unlocked: Boolean = true

    fun getCommand(): Command = command
    fun isUnlocked(): Boolean = unlocked
    fun setUnlocked(unlocked: Boolean) { this.unlocked = unlocked }
    fun getCommandId(): String = command.getId()
}
